plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -76,77 +85,68 @@ export const GroupProfileCard = ({
     ?.map(groupItem => groupItem.target.name)
     .toString();
 
-  const childrens = group?.relations
+  const childRelations = group?.relations
     ?.filter(r => r.type === RELATION_PARENT_OF)
     ?.map(groupItem => groupItem.target.name);
 
   const displayName = profile?.displayName ?? name;
+  const descriptionHeading = description ?? name;
 
   if (!group) return <Alert severity="error">User not found</Alert>;
 
   return (
     <InfoCard
-      title={<CardTitle title={displayName} />}
-      subheader={description}
+      title={<CardTitle title={descriptionHeading} />}
```

Comment:
Sure that you want to display the group description as the heading? I would expect `displayName` here? In our data the description contains a description/mission of the group, for example the field of responsibility. I think that would be quite neat as a subheader.
The `UserCard` is also doing it that way.

Reply:
Yes - difference here is the display name for, say a GitHub group called `testing-service`, is `testing-service`. WHile the display name for GitHub user `adamdmharvey` because I have it in my profile is `Adam Harvey`.

So the reason I had changed this was that the header, in most cases, just looked duplicative?

| User | Group |
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/33203301/107047199-8be8a600-6795-11eb-8415-9d99aa8106ce.png) | ![image](https://user-images.githubusercontent.com/33203301/107047100-6e1b4100-6795-11eb-9d0c-d13d8688687a.png) |

But fair, if you have very large descriptions that would wildly blow out these headers, I can see that would be problematic. All of the examples I looked at at my firm were about 5-15 words long and fit relatively well. I also checked GItHub's REST API and could not find a max size of description.

So all of that to say, happy to put this back the way it was for group card; appreciate the feedback.